app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Use Exception instead of BaseException for custom exceptions.
Add explanatory comments for each class to make the code more understandable.
Add default messages to exception class constructors for ease of use.
Use type annotations and __str__ or __repr__ for improved message output.
For example:
class NotWearingMaskError(Exception):
    """Raised when a visitor is not wearing a mask."""
    def __init__(self, message: str = "Visitor is not wearing a mask."):
        super().__init__(message)